src/components/Banner/Banner.stories.tsx

```diff
@@ -49,6 +49,7 @@ export const Brand: StoryObj<Args> = {
   render: ({ variant, ...other }) => {
     return (
       <Banner
+        // @ts-expect-error description is expected to be overridden by story args
```

Comment:
If we're requiring `description` to be present (which the `Args` type indicates), I wouldn't expect to include it here at all (since stories are presumably always supplying there own).

Or if stories don't necessarily supply their own, should `description` be an optional type on `Args`?

Reply:
ohh I didn't even notice we were typing it, will make optional